### ISSUE TITLE
Feature request for auto force stop 

### ISSUE THREAD

**Issue by TowelieTheTowel420** [d4rken-org/sdmaid-se#1177]
Would like an auto force stop for apps that are cached in background. I.e.  like apps running that aren't being used or running with like a select button to select certain apps to force stop or an app ignore list for certain apps. Would be nice

**Comment by d4rken** [comment]
Hm, not sure if in scope. SD Maid is not a "task killer". You can manually force-stop apps via AppControl already though. I don't see a good way to make this automatic though and I'm not sure whether it's a good think to add.

Are you looking for something like this?

https://play.google.com/store/apps/details?id=com.catchingnow.icebox

**Comment by TowelieTheTowel420** [comment]
No not really I just wanna force stop certain apps that are cached in background and didn't fully shut off ie like YouTube so I didn't have to open app settings to force stop. Cause it's battery usage pulling on battery in background

ImmortalTowel

Sent from Proton Mail Android

-------- Original Message --------

**Comment by MarkyMarkDE** [comment]
I would also vote for such a improvement. Some current running apps will never cleaned with sdmaid-se e.g. Playstore and the Play Services App and this cause always trouble, when you payed for an app and you always loose the premium features as you payed for. In such cases you must kill both, clear cache and sometimes their data and restart the Playstore App, then is all fine and you never lost again the payed premium for this app. Unfortunately, SDMaid SE don't note anything that such a app couldn't be cleaned, it seems that all is fine, but it isn't 😔

**Comment by MicroBeast** [comment]
Isn't "Auto" mean adding somekind of service to monitor that (continuesly)? That service alone will eat battery. Then doing its actual job in stopping apps that you use will eat battery than just let the target app Sleeping in background (for quick start etc.)

Apps like Youtube, Gplay, or Gplay Service Framework usually System Apps. Stopping apps like that is useless as they designed to always Wake Up. It means continues (try to) Stopping, eating even more battery.

The answer to no caches and no battery drain is Disable or Uninstall I believe.

**Comment by jeyjai** [comment]
This is what you're looking for
http://github.com/visnkmr/backgroundappslist

Its not automated in that app but you can choose "close multiple", select the apps you want to close.
It will automatically bring them up one after another but you still have to manually press the "force close" button and then the back button to close it, and the next will pop up

**Comment by MeloriTensei** [comment]
> @TowelieTheTowel420 This is what you're looking for [http://github.com/visnkmr/backgroundappslist](https://github.com/visnkmr/backgroundappslist)
> 
> Its not automated in that app but you can choose "close multiple", select the apps you want to close. It will automatically bring them up one after another but you still have to manually press the "force close" button and then the back button to close it, and the next will pop up

What if we can make AppControl use Accessibility Service to automate mass task kill? That should probably bring more worth to the app, right?

**Comment by Zafrkant** [comment]
Thank developer for great contribution to the community. 

I would like also to see, solution like https://gitlab.com/SuperFreezZ/SuperFreezZ, requiring accessibility to perform task, including system apps, of course if it would be inline with project future plans.

Thank you in advance.

**Comment by jeyjai** [comment]
> A "simple" solution may be to add AppControl Force stop app exclusions in the App Exclusions Manager.
> 
> ![SD Maid SE - Force stop exclusion](https://private-user-images.githubusercontent.com/34382225/397547715-8afeb833-ba6b-4283-a2a9-fde875600e24.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ3ODQ1MzMsIm5iZiI6MTczNDc4NDIzMywicGF0aCI6Ii8zNDM4MjIyNS8zOTc1NDc3MTUtOGFmZWI4MzMtYmE2Yi00MjgzLWEyYTktZmRlODc1NjAwZTI0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjIxVDEyMzAzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ4MTVhNjk0NDczNDBkMTE5OGVlMzI4MmU4N2Q5NGI0Mjk5ZWRlYTkyY2M3NTI2NGM4NjA1MmQ3YmY3ZGU2MDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VTOkhR5bXMwLyr78nbIfiDunDifms2_csWN1ZzcA3Fg)
> 
> This would make it possible to select all apps in AppControl and then Force stop apps to quickly and easily stop only unnecessary apps that are not whitelisted in the App Exclusions Manager.
> 
> ![SD Maid SE - Force stop apps](https://private-user-images.githubusercontent.com/34382225/397549712-bc4a5e60-a848-4391-a638-0ceadacad2bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ3ODQ1MzMsIm5iZiI6MTczNDc4NDIzMywicGF0aCI6Ii8zNDM4MjIyNS8zOTc1NDk3MTItYmM0YTVlNjAtYTg0OC00MzkxLWE2MzgtMGNlYWRhY2FkMmJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjIxVDEyMzAzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY4ZjEzYjJlNDRkNjdjZmY0NjYwZDAzYTU2MmM0MmRmNzE3MTAyNjAyYzFhYzQ1ZGJjMzMyOTJhMmJlZDQ4YzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.OatzCs69rgJc5ttdW19J2c08FTq_CG9tu9d2oPyGW8k)
> 
> If this can be done it could resolve this feature request and many other similar issues and discussions.

@d4rken 
Not really, most of us in here just looking for something that shows currently running processes in the background even after "closing" it by swiping it away. 
https://github.com/visnkmr/backgroundappslist

Does this exact thing and only shows the ones running in the background. That app lets you select multiple apps to close all of them at the same time by opening the app info page for each and every one app one at the time forcing you to manually click the force close button and click the back button for the next app info page for the next step open and click first close on that app.

I'm just suggesting to show a list of processes running in the background just like that https://github.com/visnkmr/backgroundappslist does and just like you use your app to clear cache by pressing the clear cash button using the accessibility setting you can do the same thing by force closing them instead.


Please try https://github.com/visnkmr/backgroundappslist and use it to close multiple apps at the same time through the button at the top and use instantly say what I mean

**Comment by d4rken** [comment]
SD Maids core purpose is freeing storage space, I don't want to position it in anyway to suggest "task-killer". While AppControl supports force-stop (ACS,ADB,ROOT), any automatic actions here are out of scope. Sorry.
